hr_contract_cae/models/hr_contract.py

```diff
@@ -0,0 +1,42 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class Contract(models.Model):
+
+    _inherit = "hr.contract"
+
+    reason = fields.Char(string="Reason for recourse to contract")
+
+    duration = fields.Integer(string="Duration", default=6)
+
+    hours = fields.Float(string="Working Hours", required=True)
+    rate = fields.Monetary(string="Hourly Rate", required=True)
```

Comment:
I would have call this hourly_wage since you use the term "wage" in other field, I think it's less confusing imho

Reply:
Good point. I thought I saw `rate` in hr.payroll, but upon closer inspection that fields means something else